### ISSUE TITLE
Re-Sturucture Honeybee Tabs to Have One for Daylight and Another for Energy

### ISSUE THREAD

**Issue by chriswmackey** [mostaphaRoudsari/Honeybee#242]

**Comment by mostaphaRoudsari** [comment]
@chriswmackey @chiensiTB I'm going to o this changes now. Wanted to make sure that you are not going to commit any changes for the next hour. Thanks!

**Comment by mostaphaRoudsari** [comment]
I made first round of changes! Next round is to separate Honeybee into two separate tabs. I'm still not sure what should I name separate tabs for Honeybee? HB_Lighting, HB_Energy or Honeybee_DL, Honeybee_E

![image](https://cloud.githubusercontent.com/assets/2915573/5895623/f3daa0ea-a4f2-11e4-8dfc-960ad941b455.png)

**Comment by mostaphaRoudsari** [comment]
Separated Honeybee components into Honeybee@DL and Honeybee@E. It still needs modifications (e.g. changing the numbers)
@chriswmackey @arkdanielnielsen  @ayezioro can you please update and let me know your thoughts on this? Thanks.

**Comment by ayezioro** [comment]
Hi Mostapha,
I understand the logic of separating DL and E+. Can agree that one tab was getting to crowded and hard to find components. Though, at the end you get used to it.
What you suggest here is more than that: separate in 3 parts, one of them embedded in the LB. This i like much less. I understand that you don't want to put the modelling section in either DL or E+, but putting it in LB is less successful in my opinion.
What about putting the modeling in the center, DL to the left and E+ to the right, keeping all together? Just thinking ...
BTW with this new layout 2 components (at least) are missing: The ItemSelector and bldPrograms, which i suspect that because they are lists they are not fixed yet for their location. Maybe there are more components like these that i didn't catch yet.

**Comment by arkdanielnielsen** [comment]
I like the divisions to get a better overview of all the different types of simulations and analysis. 
My initial comments...
1) I agree with @ayezioro about the model components that is placed under LB is not the most logical placement.
2) What I originally suggested was tabs for Daylight/Lighting, Energy/Comfort and a other for modelling and import/export models. 
3) It could also be useful to split the 09|Energy|Energy in smaller sections, like Simulation, Read/Import, Visualization, Analysis (further analysis) sections!? The same order could be used for the @DL.
4) A other challenge is many of the visualization and Legends components works both for the weather data and daylight/energy components. This could also be more clear.
5) 09|Energy|Energy: components like Context, Shading, and Adiabatic should maybe be placed together with the model components, because some of them also work for the @DL simulation?
6) Maybe the Honeybee@E should have a C as well because of all the comfort capabilities?

**Comment by mostaphaRoudsari** [comment]
Hi Abraham and Daniel,

Thank you for the feedback and also for the missed components!

Abraham, the main reason to do this separation is to make it easier to access the components from the tabs. Don't you think that there is so much under Honeybee tab right now?

I think I have to agree with you that having HB modeling components under LB is not the most intuitive approach. I also have to agree with almost all the other comments except I don't really want to create 4 different tabs. What about having modeling components and DL components under the same tab and move energy and comfort components to a separate tab. So we get Ladybug, Honeybee and Honeybee@SomethingThatIDontKnow. (I personally find EC confusing unless I put as E+C which is then long!)

If you think 3 tabs as I mentioned above is cool then I can go ahead and give it another try tonight.

**Comment by arkdanielnielsen** [comment]
What about having HB1 for the modeling and material/construction components - the first step prepare model for simulation, and then going to HB2 - step 2, with the components for simulation and analysis... but there will probably be a few more components in HB2. Maybe some of the comp. from LB could be re-organized as well like the shading designer. I have always though they did not really fitted in between the other ones in LB. 
It is not optimal either the modeling+DL and E+C or the suggestion above. 
I dont no... it is a challenge....

**Comment by ayezioro** [comment]
Hi,
I think that one more option is leave it "as is". This was a long day for me so i'll juast write the reason: Cognitive.  Users probably have installed more plugins. Each with its own tab. When GH starts happens that the order of those tabs is kind of random. I can imagine it can be kind of anoying each time looking for the tabs. Moreover, i can't think if every plugin will consider splitting their developments in many tabs.
In short i'm thinking a bit broader at the whole GH tab world.
Two cents ...
Hard decision.
-A.

**Comment by chriswmackey** [comment]
Hello @mostaphaRoudsari , @arkdanielnielsen and @ayezioro ,

Sorry for coming late to the discussion.  I had also initially thought that three Honeybee tabs would be a better organization (setup, daylight, and energy) but received push back that 4 tabs was a lot.  Given that Honeybee now effectively replaces two other tabs that I used to have (DIVA and ArchSim), I don't have much of an issue with more tabs but I guess it might be a bit selfish to take up too much GH real estate.

For Some Minor Tweaks that I Would Suggest if the Current Setup Stays Around:
@mostaphaRoudsari , as per our discussion, I was originally thinking that the honeybee modeling components would be added to the 5 section of Ladybug where the Honeybee cocoon had been (it seems a bit preemptive and possibly confusing to new users to add them to the very first section of Ladybug).  I also do not know how the @ symbol is meant to be used, which I always understood as meaning "at".  It seems weird to say "Honeybee at Daylight" (perhaps just a Honeyee-DL would be good).  Also, maybe having the HB tabs abbreviate Honeybee instead of Daylight or energy (like HB-Energy and HB-Daylight) might be a bit clearer.

For My General Feelings Now that I see the Setup:
Now that I see it, I can't help feeling that this reorganization based on Energy and Daylight is not right and that we should do something that sticks more to the original arrangement (half-agreeing with Abraham).  I understand why we need to do something if we want Honeybee to continue to grow but having it take over the Ladybug tab seems to be potentially dangerous from the perspective of a new user (now that I see it, I feel like I am going to have to re-make the Intro to Ladybug video tutorial series because new users will get confused by the arrangement).  I think that it's a better idea to go with Daniel's suggestion of a "Honeybee1" tab and a "Honeybee2" tab.  I am ok with Honeybee1 just being the setup or it being both the setup and the daylight.  Both make much more sense to me than the current arrangement that just seems to be mixing Ladybug and Honeybee too much.  It really feels like we made a new plugin called Ladybee or Honeybug.

-Chris

**Comment by mostaphaRoudsari** [comment]
Thank you all for the comments. I agree with your comments and I think it is a good idea to give us a little bit more time and get back to this issue again. Looks like there is no single solution to address everything. I think it is a valid concern to keep the number of tabs as less as possible do if we can make it happen by just two tabs then we should do it with two.

I reverted back the changes so if you update now you should have the version with two tabs back.

I remove this from the changes for this release but keep it open so I can address some of the minor modifications that you guys suggested.

**Comment by mostaphaRoudsari** [comment]
This was a bad idea! Thank you for not letting us doing this.
